src/main/java/org/openstreetmap/atlas/utilities/command/subcommands/AtlasDiffCommand.java

```diff
@@ -23,12 +28,17 @@
 import org.openstreetmap.atlas.utilities.command.parsing.ArgumentArity;
 import org.openstreetmap.atlas.utilities.command.parsing.ArgumentOptionality;
 import org.openstreetmap.atlas.utilities.command.parsing.OptionOptionality;
+import org.openstreetmap.atlas.utilities.command.terminal.TTYAttribute;
+
+import com.google.common.collect.Sets;
 
 /**
  * @author lcram
  */
 public class AtlasDiffCommand extends AbstractAtlasShellToolsCommand
 {
+    static final String NO_CHANGE = "Atlases are effectively identical";
+
     private static final String BEFORE_ATLAS_ARGUMENT = "before-atlas";
     private static final String AFTER_ATLAS_ARGUMENT = "after-atlas";
```

Comment:
Can you change the string values here to `before-atlas(es)`/`after-atlas(es)` since we now support directories? It will make the documentation more clear.